docs/data/sdks/android/index.md

```diff
@@ -155,6 +155,24 @@ Accurate session tracking requires that you enable `enableForegroundTracking(ge
     val sameClient = Amplitude.getInstance("Andy_Client")
     ```
 
+#### Configuration
+
+???config "Configuration Options"
```

Comment:
Is `???` intentional?

Reply:
yeah, that's the format part I mentioned. This would show an expandable section other than a plain table.